cdap-data-fabric/src/main/java/co/cask/cdap/data2/util/hbase/HBaseTableUtil.java

```diff
@@ -385,6 +388,73 @@ public OutputStream getOutput() throws IOException {
     return info;
   }
 
+  protected String prefixCDAPToNamespace(String namespace) {
```

Comment:
I would name this getHBaseNamespace() or toHBaseNamespace() to more clearly identify what it represents.